frontend/packages/knative-plugin/src/components/revisions/__tests__/RevisionRow.spec.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import * as _ from 'lodash';
+import { TableData } from '@console/internal/components/factory';
+import { ResourceLink } from '@console/internal/components/utils';
+import { K8sResourceConditionStatus } from '@console/internal/module/k8s';
+import { revisionObj } from '@console/dev-console/src/components/topology/__tests__/topology-knative-test-data';
```

Comment:
It is wrong dependency. knative-plugin should have no dependency on dev-console packages. Not for this PR but create a task to fix this dependency issue.

Reply:
We do have this dependency right now. Knative Plugin depends on dev-console to render stuff in topology and a lot of other such instances. We probably need to come up with a runtime extensibility framework to fully eliminate the dependency.